apps/emqx/src/emqx_persistent_session_ds.erl

```diff
@@ -1182,7 +1182,12 @@ maybe_set_offline_info(S, Id) ->
     case emqx_cm:lookup_client({clientid, Id}) of
         [{_Key, ChannelInfo, Stats}] ->
             emqx_persistent_session_ds_state:set_offline_info(
-                #{chan_info => ChannelInfo, stats => Stats},
+                #{
+                    chan_info => ChannelInfo,
```

Comment:
I remember that there were some discussions that channel info includes unnecessary data like certificates. Can you verify that this is not included here?

Reply:
Luckily, it's not contained here:

<details> <summary> `ChannelInfo` </summary>

```erlang
{'emqx@127.0.0.1',emqx_persistent_session_ds,1184}>>>>>>>>>
  #{ci =>
        #{clientinfo =>
              #{protocol => mqtt,username => undefined,
                clientid => <<"testds">>,user_property => #{},
                is_superuser => false,mountpoint => undefined,
                listener => 'ssl:default',zone => default,
                enable_authn => true,
                peerhost => {127,0,0,1},
                sockport => 8883,is_bridge => false,peerport => 37746,
                client_attrs => #{},auth_expire_at => undefined},
          conninfo =>
              #{peername => {{127,0,0,1},37746},
                sockname => {{127,0,0,1},8883},
                keepalive => 60,username => undefined,
                clientid => <<"testds">>,expiry_interval => 7200000,
                clean_start => false,connected_at => 1715630053096,
                proto_ver => 4,proto_name => <<"MQTT">>,receive_maximum => 32,
                conn_mod => emqx_connection,socktype => ssl,conn_props => #{}},
          session =>
              #{id => <<"testds">>,
                subscriptions =>
                    #{<<"ds/t">> =>
                          #{id => 0,
                            subopts =>
                                #{nl => 0,qos => 2,rh => 0,rap => 0,
                                  sub_props => #{}},
                            current_state => 1,start_time => 1715630053098}},
                created_at => 1715630053094,retry_interval => 30000,
                await_rel_timeout => 300000,upgrade_qos => false,
                is_persistent => true},
          conn_state => connected,will_msg => undefined,
          sockinfo =>
              #{peername => {{127,0,0,1},37746},
                sockname => {{127,0,0,1},8883},
                socktype => ssl,sockstate => running}}}
```

</details>